weave

```diff
@@ -53,7 +53,8 @@ weave dns-lookup    <unqualified_name>
 weave expose        [<addr> ...] [-h <fqdn>]
 weave hide          [<addr> ...]
 weave ps            [<container_id> ...]
-weave status
+weave report        [-f <format>]
+weave status        [peers | connections | dns]
```

Comment:
I'd switch those around - in the code too - since `status` is the one we would expect most users to invoke.